### DISCUSSION TITLE
How to see HTTP in browser dev tools

### DISCUSSION THREAD

**Discussion by dfberry** [tokio-rs/axum#2620] [Q&A]
### Summary

Bare bones Axum - just learning this framework. 

When I make a request to Axum server locally from a browser, I don't see the HTTP request and response as I would in other languages and frameworks. Instead I see, `viewer.css` and `web-accessible.js`. How do I set up axum to see the HTTP traffic in the browser's devtools?

### axum version

0.7.4

**Comment by jplatte** [comment]
No such files exist in axum. This has to be related to your specific application, not axum.

**Reply by dfberry** [reply]
Ah - it was a chrome extension I was unaware of. That you for responding so quickly.
